website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -112,10 +114,14 @@ When project-scoped variable sets have conflicting variables, Terraform Cloud co
 
 For example, if you apply `A_Variable_Set` and `B_Variable_Set` to the same project, Terraform Cloud uses any conflicting variables from `A_Variable_Set`. This is the case regardless of which variable set has been edited most recently. Terraform Cloud only considers the lexical ordering of variable set names when determining precedence.
 
+If prioritized [prioritized](/terraform/cloud-docs/workspaces/variables/index#precedence-with-priority-variable-sets), variables in a project variable set have precedence over project and workspace scoped variable sets, workspace applied, and run applied variables with the same key.
+
 ### 6. Global Variable Sets
 
 Workspace and project-scoped variable sets always take precedence over global variable sets that are applied to all workspaces within an organization. Terraform does not allow global variable sets to contain variables with the same key, so they cannot conflict.
 
+If prioritized [prioritized](/terraform/cloud-docs/workspaces/variables/index#precedence-with-priority-variable-sets), variables in a global variable set have precedence over all other variables with the same key.
```

Comment:
because of the ordering in the list, i think adding the prioritized note in the addendum actually makes this more confusing. sorry if i wasn't clear on that one! i think updating this entire list to match the diagram change you made and reflect what sarah wrote [here](https://hashicorp.slack.com/archives/C05RVL6NVCK/p1698174139268749?thread_ts=1698167987.050539&cid=C05RVL6NVCK) would be clearest.